xcode/Subconscious/Shared/Models/Slashlink.swift

```diff
@@ -206,6 +206,18 @@ extension Slashlink {
         }
     }
     
+    /// Rebase one Slashlink on another.
+    /// If `@alice.bob/test` appears in a note owned by `@donna.charlie`
+    /// then we want `@alice.bob.donna.charlie/test`
+    func rebaseIfNeeded(slashlink: Slashlink) -> Slashlink {
```

Comment:
This is trying to say: If the slashlink `self` is embedded within a note at the address `slashlink` then what is the full address?

I think the signature is a bit confusing at the moment but it's an important operation